src/components/widgets/HiddenWidget.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from "react";
+
+
+function HiddenWidget({
+  schema,
+  id,
+  placeholder,
+  value,
+  defaultValue,
+  required
+}) {
+  return (
+    <input type="hidden"
+      id={id}
+      value={value}
+      defaultValue={defaultValue}
+      required={required} />
```

Comment:
This should be removed. Maybe we could add a test proving that validation works an hidden field is marked as required in the schema definition.